net-analyzer/greenbone-security-assistant/greenbone-security-assistant-21.4.4.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
I have a slight preference to name this package `gsad` and not `greenbone-security-assistant`, as it makes writing and reading the ebuild easier.